modules/calib3d/src/fundam.cpp

```diff
@@ -944,7 +948,7 @@ void cv::convertPointsFromHomogeneous( InputArray _src, OutputArray _dst )
             Point2f* dptr = dst.ptr<Point2f>();
             for( i = 0; i < npoints; i++ )
             {
-                float scale = sptr[i].z != 0 ? 1.f/sptr[i].z : 1.f;
```

Comment:
`sptr[i]` is `int` here, we can compare it with zero safely.

Reply:
@mshabunin i believe `sptr[i]` is `float` from here
https://github.com/opencv/opencv/blob/master/modules/calib3d/src/fundam.cpp#L966

Reply:
Oh, yes, sorry. Will revert this and the other similar line back.

Reply:
reverted mistakenly changed lines where the value type was integer